arcs-long functionality (#98)
bcgsc/arcs: README.md

# ARCS

Scaffolding genome sequence assemblies using 10X Genomics GemCode/Chromium data. 
ARCS can be run in 3 modes:
* [ARCS](https://doi.org/10.1101/100750) (default) uses alignments of linked reads to the input contigs

> * ARCS-long (`arcs-long`) uses alignments of long reads to the input contigs (only available when running the ARCS+LINKS pipeline) 

Reviewer: ```suggestion
* ARCS-long (`arcs-long`) uses alignments of long reads to the input contigs (only available when running ARCS in default mode) 
```
Just because people might be confused, since `ARKS` and `ARCS` are all in this one repo now

# ARCS

Scaffolding genome sequence assemblies using linked or long read sequencing data. 

> ARCS can be run in 3 modes:

Reviewer: ```suggestion
ARCS can be run in 4 modes:
```

The ARCS+LINKS pipeline requires two input files:
* Draft assembly fasta file

> * Reads file in fasta format `*.fa.gz` (or fastq format `*.fq.gz` if using long reads)

Reviewer: I think you mean the opposite? Since short reads are normally in `.fq.gz` format?

To run the `arcs` executable in default mode, run `arcs <alignments>`. For descriptions of all arguments, run `arcs --help`.

### Running ARCS in '--arcs-long' mode

> The arcs-long mode uses alignments of long reads to scaffold the input contigs.

Reviewer: ```suggestion
The arcs-long mode first simulates pseudo-linked reads from long reads, then uses alignments of the pseudo-linked reads to scaffold the input contigs.
```
Or something similar? Thinking just so they don't think we align the full long reads

To run the pipeline in arcs-long mode, run `Examples/arcs-make arks-long`. For example, to scaffold the assembly `my_scaffolds.fa` with long reads `my_reads.fa.gz` or `my_reads.fq.gz`, specifying a minimum contig length of 1000bp:
```
arcs-make arcs-long draft=my_scaffolds reads=my_reads z=1000
```

For more info check `Examples/arcs-make help`.

> **Parameters**: To account for the higher error rates in long reads vs long reads, we suggest starting with the following values: 

Reviewer: ```suggestion
**Parameters**: To account for the higher error rates in long reads vs linked reads, we suggest starting with the following values: 
```

To run the `arcs` executable in ARKS mode, run `arcs --arks`. For descriptions of all arguments, run `arcs --help`.

### Running ARCS in '--arks-long' mode

The arks-long mode first segments and assigns barcodes to the long reads, yielding pseudo-linked reads. Scaffolding is performed based on exact k-mer mapping of pseudo-linked reads to the input contigs.

> To run the pipeline in arks-long mode, run `Examples/arcs-make arks-long`. For example, to scaffold the assembly `my_scaffolds.fa` with long reads `my_reads.fa.gz` or `my_reads.fq.gz`, specifying a kmer size of 20:

Reviewer: ```suggestion
To run the pipeline in arks-long mode, run `Examples/arcs-make arks-long`. For example, to scaffold the assembly `my_scaffolds.fa` with long reads `my_reads.fa.gz` or `my_reads.fq.gz`, specifying a kmer size of 20 and `j` of 0.05:
```
Just thinking to also mention the `j` that we suggest they use to drive that setting home.

```

> arcs-make arks-long draft=my_scaffolds reads=my_reads k=20

Reviewer: ```suggestion
arcs-make arks-long draft=my_scaffolds reads=my_reads k=20 j=0.05
```
